test/wh_test_crypto.c

```diff
@@ -367,6 +397,7 @@ int whTest_CryptoClientConfig(whClientConfig* config)
         printf("RSA SUCCESS\n");
     else
         printf("RSA FAILED TO MATCH\n");
+#endif
```

Comment:
Let's add closing comments for long (more than 10 lines) macro blocks. Makes it easier to keep track of, especially when nested.

```suggestion
#endif /* !NO_RSA */
```